assets/robotest/run.sh

```diff
@@ -25,7 +25,7 @@ GCE_REGION="northamerica-northeast1,us-west1,us-east1,us-east4,us-central1"
 GCE_VM=${GCE_VM:-custom-4-8192}
 GCE_PREEMPTIBLE=${GCE_PREEMPTIBLE:-'false'}
 # Parallelism & retry, tuned for GCE
-PARALLEL_TESTS=${PARALLEL_TESTS:-4}
+PARALLEL_TESTS=${PARALLEL_TESTS:-10}
```

Comment:
Just keep an eye out for us hitting our limits, I think we were pretty close the other day in a couple of regions.

Reply:
Will do.  I haven't seen limit related failures yet, but robotest certainly isn't smart about allocation currently -- it just selects randomly from a list.